apps/chat-web/app/components/alert.tsx

```diff
@@ -0,0 +1,28 @@
+import { twMerge } from 'tailwind-merge'
+import WarnIcon from './icons/warn-icon'
+import ErrorIcon from './icons/error-icon'
+
+export interface AlertProps {
+  message: string
+  type: 'warning' | 'error'
+  children?: JSX.Element
+}
+const Alert = ({ message, type, children }: AlertProps) => {
+  return (
+    <div
+      role="alert"
```

Comment:
The Alert container is too large. Maybe this fix:

```suggestion
      role="alert p-2"
```